### DISCUSSION TITLE
Welcome to VoiceCraft-MCBE_Proximity_Chat Discussions!

### DISCUSSION THREAD

**Discussion by SineVector241** [AvionBlock/VoiceCraft#100] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SineVector241** [comment]
I am currently the only maintainer.

**Reply by javier071096** [reply]
hola, podrías renovar el certificado por favor me sale que ya ha expirado
